AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.html

```diff
@@ -78,7 +78,7 @@
         </td>
         <td class="govuk-table__cell">{{ hearing.hearing_date.toLocaleDateString('en-GB') }}</td>
         <td class="govuk-table__cell">{{ hearing.start_time }}</td>
-        <td class="govuk-table__cell">{{ hearing.duration }}</td>
+        <td class="govuk-table__cell">{{ dateAndTimeService.minutesToHoursDisplay(hearing.duration) }}</td>
```

Comment:
Now that I've seen it in action, this is a great candidate for you create a pipe operator

```suggestion
        <td class="govuk-table__cell">{{ hearing.duration | HoursAndMinutes }}</td>
```

[see create pipes for data transformations](https://angular.io/guide/pipes#creating-pipes-for-custom-data-transformations)

Reply:
Yeah good suggestion!